mappings/net/minecraft/entity/projectile/ProjectileEntity.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_1676 net/minecraft/entity/projectile/ProjectileEntity
 	FIELD field_22478 ownerUuid Ljava/util/UUID;
+	FIELD field_23739 ownerEntityId I
+	FIELD field_23740 leftOwner Z
```

Comment:
What does this field do?

Reply:
`ownerEntityId` is the entity ID of the owner, matching `ownerUuid`.

From looking at `method_26961`, it looks like `leftOwner` (name from its corresponding NBT tag) is whether or not the projectile is out of a range from its owner.

Reply:
Yeah otherwise mojang projectile have like hitting owner/blocking sight in rendering etc